stable-2.4.0.md

```diff
@@ -0,0 +1,188 @@
+## stable-2.4.0
+
+This stable release introduces adds support for SMI [Traffic Split](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md),
+fine-grained observability into the TLS-based identity system via the new
+`linkerd edges` command, default automatic proxy injection, installation stages,
+and a host of performance and usability improvements in the proxy and Web UI.
+
+Creating an SMI [Traffic Split](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md)
+resource will allow Linkerd to split traffic between the specified backend
+services. The `linkerd edges` command provides a thorough breakdown of the name
+and identity for proxied connections, as well as an exciting new "Edges" table
```

Comment:
"The `linkerd edges` command provides a thorough breakdown of the name
and identity for proxied connections"
That sounds like the connections are named and identified, which might not be totally accurate. Probably needs a rephrase like ".. provides a thorough breakdown of the proxied connections, with the names and identity of each resource on both sides of the connection"